functional/tests/health-check.test.mjs

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+import { expect, jest, test, beforeAll, afterAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(20010);
+
+const httpPort = 3456;
+const socketPort = 9876;
+
+beforeAll(async () => {
+  console.log(
+    await utilities.mfState(
+      true,
+      ` -- --httpPort ${httpPort} --socketPort ${socketPort}`
+    )
+  );
+});
+
+afterAll(async () => {
+  // to kill the Mock Firebolt
+  console.log(await utilities.killPort(socketPort));
+});
+
+test(`MF Startup/Health Check in a different port`, async () => {
+  let healthCheckResponse = await utilities.callApi(
+    "/api/v1/healthcheck",
+    "",
+    "",
+    "GET",
+    httpPort
+  );
+  expect(healthCheckResponse).toBeDefined();
+  expect(healthCheckResponse.status).toBe(200);
+});
+
+test(`run MF cli command to perform heath check in a different port`, async () => {
```

Comment:
"different" can be changed to "custom" ?

Reply:
sure. Let me do that.